docs/src/layouts/MainLayout.vue

```diff
@@ -1,74 +1,80 @@
+<!--            />-->
             <q-btn
-              color="lp-light"
-              flat
-              label="Services"
-            />
-            <q-btn
-              color="lp-light"
+              :color="$route.name === 'lpComponents'? 'white' : 'lp-light'"
               flat
               label="Components"
               class="q-py-sm q-px-md"
               :to="{ name: 'lpComponents' }"
+              size="16px"
             />
             <q-btn
-              color="lp-light"
+              :color="$route.name === 'sponsors'? 'white' : 'lp-light'"
               flat
               label="Become sponsor"
               class="q-py-sm q-px-md"
+              size="16px"
+              to="/sponsors-and-backers"
             />
             <q-btn
-              color="lp-light"
+              :color="$route.name === 'about'? 'white' : 'lp-light'"
               flat
               label="About"
               class="q-py-sm q-px-md"
+              size="16px"
             />
             <q-btn
-              color="lp-light"
+              :color="$route.name === 'blog'? 'white' : 'lp-light'"
               flat
               label="Blog"
+              size="16px"
             />
           </div>
-          <q-btn flat round color="lp-primary" icon="search"/>
+          <q-btn flat round color="lp-primary" icon="search" size="16px"/>
         </div>
 
       </q-toolbar>
       <q-separator color="lp-primary"/>
-      <div class="row justify-end shadow-bottom-small q-py-xs q-pr-md social-links">
-        <q-btn label="Why Quasar?" flat color="lp-light" no-caps size="sm"/>
-        <q-btn label="Getting Started" flat no-caps color="lp-light" size="sm"/>
-        <q-btn label="Roadmap" flat no-caps color="lp-light" size="sm"/>
-        <q-btn label="Video Tutorials" flat no-caps color="lp-light" size="sm"/>
-        <q-btn label="Quasar Brand resources" flat no-caps color="lp-light" size="sm"/>
+      <div class="row justify-end q-py-xs q-pr-md social-links shadow-bottom-small">
+        <q-btn label="Why Quasar?" flat color="lp-light" no-caps size="12px" to="/introduction-to-quasar"/>
+        <q-btn label="Getting Started" flat no-caps color="lp-light" size="12px" to="/start/pick-quasar-flavour" />
+        <q-btn label="Roadmap" flat no-caps color="lp-light" size="12px" to="/start/roadmap"/>
+        <q-btn label="Video Tutorials" flat no-caps color="lp-light" size="12px" to="/video-tutorials" />
+        <q-btn label="Quasar Brand resources" flat no-caps color="lp-light" size="12px" type="a" href="https://github.com/quasarframework/quasar-art" target="__blank" />
         <template v-for="(socialLink, linkIndex) in socialLinks" :key="linkIndex">
-          <q-btn :icon="socialLink.icon" flat color="lp-primary" round size="sm" type="a" :href="socialLink.href" target="__blank"/>
+          <q-btn :icon="socialLink.icon" flat color="lp-primary" round size="md" type="a" :href="socialLink.href" target="__blank"/>
```

Comment:
`.reverse()` on `socialLinks` mutates the original array, when you do it for the footer links you revers top links too
Also, the Discord icon is wrong